### DISCUSSION TITLE
Custom title not always displayed

### DISCUSSION THREAD

**Discussion by allesoverwaterstof** [JulianPrieber/littlelink-custom#334] [Issues]
Why is the custom title not always displayed when filled. Eg with 'default email' it is not shown.

**Comment by JulianPrieber** [comment]
It seems we have a problem in our internal system. I'm gonna look into it.

**Reply by JulianPrieber** [reply]
Got a fix for it ready: https://github.com/JulianPrieber/littlelink-custom/commit/cc7c03dab560a6b1c93c87a05a615560a84fc234

**Reply by allesoverwaterstof** [reply]
thats fast! wow.. Works great tnx
